src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,23 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo';
+
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => {
+  if (comments.length === 0) {
+    return (
+      <>
+        <hr />
+
+        <b data-cy="NoCommentsMessage">No comments yet</b>
+      </>
+    );
+  }
```

Comment:
You shouldn't render CommentList component at all if there are no comments, so move this logic to the parent component